### ISSUE TITLE
https for style.css

### ISSUE THREAD

**Issue by LDAV** [wpovernight/woocommerce-pdf-invoices-packing-slips#25]
Should be better if you change these notations in /woocommerce/pdf/Simple/style.css
from:
`src: local('Open Sans'), local('OpenSans'), url(http://themes.googleusercontent.com/...`
to:
`src: local('Open Sans'), local('OpenSans'), url(//themes.googleusercontent.com/...`

This should prevent non https warning in https sites.

**Comment by Spreeuw** [comment]
These are not frontend urls, they are used by dompdf on the server - this means it doesn't really matter that much because there's no browser involved. More importantly, Open Sans is bundled with the plugin fonts by default. In the past I left this in the CSS file as a hint for anyone who wanted to work with custom fonts. But over the years I learned that using URLs is not the best universal method here because many servers have `allow_url_fopen` set to false for security.

Long story short: I will remove this font declaration entirely and put a link in the CSS to the documentation about custom fonts.

Thanks for the heads up!
